modules/rules_perl/0.2.4/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "integrity": "sha256-vHVQ84gfYyVaZcbex86Dw14K5I17mqfm6SN7BnM8v/g=",
+    "strip_prefix": "rules_perl-0.2.4",
+    "url": "https://github.com/bazel-contrib/rules_perl/archive/refs/tags/0.2.4.tar.gz",
```

Comment:
It'll be nice if you can actually create a release archive.

Reply:
I'm not a maintainer of that repo